### PR TITLE
chore: upgrade rusty_v8 to 0.55.0

### PR DESCRIPTION
**Pull request by bartlomieju**

### CODE REVIEW

**Review by dsherret — APPROVED** [review]
LGTM

**Review by littledivy — COMMENTED** [review]
`--experimental-wasm-threads` is a no longer a flag?

File: cli/napi/js_native_api.rs [hunk 1]
```diff
@@ -1260,10 +1260,11 @@ fn napi_delete_reference(env: *mut Env, _nref: napi_ref) -> Result {
 }
 
 #[napi_sym::napi_sym]
-fn napi_detach_arraybuffer(_env: *mut Env, value: napi_value) -> Result {
+fn napi_detach_arraybuffer(env: *mut Env, value: napi_value) -> Result {
+  let env: &mut Env = env.as_mut().ok_or(Error::InvalidArg)?;
   let value = transmute::<napi_value, v8::Local<v8::Value>>(value);
   let ab = v8::Local::<v8::ArrayBuffer>::try_from(value).unwrap();
-  ab.detach();
+  ab.detach(v8::undefined(&mut env.scope()).into());
```
Comment (littledivy) [inline cli/napi/js_native_api.rs:1267]: we can pass nullable maybe handles too right? I think rusty_v8 should accept Option

Comment (bartlomieju) [reply cli/napi/js_native_api.rs:1267]: That's what Andreu suggested in https://github.com/denoland/rusty_v8/pull/1118#discussion_r1019726816 but I had no time to look into it. Feel free to open a PR

**Review by littledivy — APPROVED** [review]

**Pull request merged**